readthedocs/api/v2/views/footer_views.py

```diff
@@ -81,6 +81,19 @@ class BaseFooterHTML(APIView):
     """
     Render and return footer markup.
 
+    Query parameters:
+
+    - project
+    - version
+    - page: Sphinx's page name, used for path operations,
+      like change between languages (deprecated in favor of ``origin``).
+    - origin: Full path with domain, used for path operations.
```

Comment:
Why don't we call it `full_path`?

Reply:
or `absolute_uri` as Django calls it when it has the domain on it.

Reply:
I'm avoiding naming this path since it includes the domain as well, we use full path to refer to a path in other parts of the code. Not sure about uri

Reply:
Yea, origin isn't the right name, since that's used in HTTP for just the origin hostname: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin

I think `absolute_uri` or similar is best: https://docs.djangoproject.com/en/3.1/ref/request-response/#django.http.HttpRequest.build_absolute_uri